### ISSUE TITLE
Not loading presets on launch

### ISSUE THREAD

**Issue by DartRuffian** [BrettMayson/HEMTT#600]
HEMTT does not load mods from `presets` when launching. If the Steam Workshop ids are placed in `workshop` they (down)load and are added to the startup parameters like normal. [Veteran29 also replicated the issue on their end](https://ptb.discord.com/channels/976165959041679380/1039189045655392389/1171219404164452515).

---

HEMTT Version: `1.10.0`
[HTML Modlist (Pastebin)](https://pastebin.com/DVkvLADK). Located at `.hemtt\presets\main.html`
Launch Properties:
```toml
[hemtt.launch.default]
workshop = []
presets = [
    "main"
]
dlc = []
optionals = []
parameters = []
```

Launching with `hemtt.exe launch`.

**Comment by BrettMayson** [comment]
Please try again on 1.10.1, some fixes related to HTML presets were included.

**Comment by DartRuffian** [comment]
Updated to 1.10.1, tried both `presets = ["main"]` and `main.html`, neither case loaded the mods listed in the preset.

**Comment by BrettMayson** [comment]
Very odd, I added your muffler as a test case and it passes just fine

**Comment by DartRuffian** [comment]
How should it be set in `project.toml`?
Could be me just setting it incorrectly

**Comment by BrettMayson** [comment]
What you posted looks fine, adding the .hemttout/latest.log would be great too

**Comment by DartRuffian** [comment]
[.hemttout/latest.log](https://pastebin.com/bk1FgCHZ)
Only other change was adding an additional mod to the `main.html` file (done through the launcher and re-exported).

**Comment by BrettMayson** [comment]
I've sent you a build on discord if you could test with that and add the latest.log here

**Comment by DartRuffian** [comment]
[latest.log](https://github.com/BrettMayson/HEMTT/files/13445223/latest.log) using build with extra logging.
> if `workshop.is_empty()` is correct
> I had if `launch.workshop.is_empty()` by mistake
> \- Brett

Work-around is to include a single mod in `workshop`.

**Comment by BrettMayson** [comment]
Fixed on main, will be in 1.10.2
